core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/workflow/WorkflowExecutionsResource.scala

```diff
@@ -54,6 +55,15 @@ object WorkflowExecutionsResource {
     .createDSLContext()
   final private lazy val executionsDao = new WorkflowExecutionsDao(context.configuration)
 
+  val statusStringToCode: Map[String, Byte] = Map(
```

Comment:
It's better to put this in `src/main/scala/edu/uci/ics/amber/engine/common/Utils.scala` as we already have similar conversion functions. Try to reuse them if you can. If not, explicitly mention this is from frontend status to byte status code in the method name.

We currently have 3 representations of execution status. 1) frontend: string. 2) backend: AggregatedState, 3) DB: byte. We should unify them in the future.

Reply:
Fixed